navigation-breadcrumb/nav-breadcrumb.html

```diff
@@ -0,0 +1,30 @@
+<!-- Command line prompt $ python -m SimpleHTTPServer 8000 -->
```

Comment:
Is this necessary since this is just HTML + CSS for now?

Reply:
Axe-core requires a file be served to evaluate, but not needed otherwise. Can remove.